Lib/test/test_ssl.py

```diff
@@ -3867,7 +3867,7 @@ def test_min_max_version_mismatch(self):
                                             server_hostname=hostname) as s:
                 with self.assertRaises(ssl.SSLError) as e:
                     s.connect((HOST, server.port))
-                self.assertRegex("(alert|ALERT)", str(e.exception))
+                self.assertRegex(str(e.exception), "(alert|ALERT)")
```

Comment:
https://github.com/python/cpython/pull/117351 will take care of this on its own.

Reply:
thanks @gpshead.